docs/src/examples/01-loading-and-saving.jl

```diff
@@ -102,3 +109,12 @@ save_model(model_converted_to_json, "e_coli_core_from_sbml.json")
 println(open("e_coli_core_from_sbml.json") do f
     read(f, 100)
 end |> String, "...")
+
+# You can also serialize models for quicker access, but this is moderately
+# dangerous because Julia offers no guarantees that different machines will be
+# able to deserialize datastructures. 
+
+using Serialization
+
+serialize("model_json.js", model)
+model = deserialize("model_json.js")
```

Comment:
I'd honestly leave this out. The logic is:
- this automagickally saves everything and for beginners this has always inevitably lead to the "matlab workspace from 1981" problem
- non-beginners will know this
- advanced people will know why it's a bad idea

Reply:
okay, removing!